bin/tag-generator.py

```diff
@@ -4,14 +4,17 @@
 ALPINE_VERSIONS = ['3.16', '3.17', '3.18', '3.19']
 
 LTS_VERSIONS = [ "8", "11", "17", "21"]
-
+UNSUPPORTED_AL2_VERSIONS = [ "22" ] 
 def generate_tags(key, version):
     update = version.split('.')[1] if (key == '8') else version.split('.')[2]
     expanded_version = f"{key}u{update}" if (key == '8') else f"{key}.0.{update}"
 
-    al2_tags = [f"{key}", f"{expanded_version}", f"{expanded_version}-al2", f"{key}-al2-full", f"{key}-al2-jdk", f"{key}-al2-generic", f"{expanded_version}-al2-generic", f"{key}-al2-generic-jdk"]
     al2023_tags = [f"{key}-al2023",  f"{expanded_version}-al2023" ,f"{key}-al2023-jdk"]
     al2023_generic_tags = [f"{key}-al2023-generic", f"{expanded_version}-al2023-generic", f"{key}-al2023-generic-jdk"]
+    if key not in UNSUPPORTED_AL2_VERSIONS:
```

Comment:
No new versions will be supported on AL2, so lets change the logic so we don't have to add 23 here in 5 months.